tools/check_format_test_run_under_docker.py

```diff
@@ -4,10 +4,16 @@
 # check_format_test.py, or you are liable to get the wrong clang, and
 # all kinds of bad results.
 
+import argparse
 import os
 import shutil
 import subprocess
 
+loglevel = 1
+def logMessage(level, message):
+  if level < loglevel:
```

Comment:
Should we use Python `logging` (https://docs.python.org/3/library/logging.html)?

Reply:
done